tests/rest/client/test_rooms.py

```diff
@@ -3166,3 +3185,21 @@ def test_threepid_invite_spamcheck(self) -> None:
 
         # Also check that it stopped before calling _make_and_store_3pid_invite.
         make_invite_mock.assert_called_once()
+
+        # Run variant with `Tuple[Codes, dict]`.
+        mock.return_value = make_awaitable((Codes.CONSENT_NOT_GIVEN, {}))
```

Comment:
We should test that the response contains the right code and fields (same with other similar tests).